.github/workflows/run-tests-windows.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [14.x]
```

Comment:
it would be better to set concrete Node version `14.17.5`

Reply:
fixed.